manticore/platforms/linux.py

```diff
@@ -1753,30 +1716,23 @@ def sys_exit_group(self, error_code):
         self.sched()
         self.running.remove(procid)
         #self.procs[procid] = None
-        logger.debug("EXIT_GROUP PROC_%02d %s", procid, ctypes.c_int32(error_code).value)
         if len(self.running) == 0 :
```

Comment:
with this, sys_exit no longer prints out a log because that syscall implementation bails with an exception and doesn't return. this might be fine though, since we get a testcase generated msg immediately after 

```
2017-11-28 17:25:38,376: [37716] m.c.executor:INFO: Forking, about to store. (policy: ALL, values: 0x400a9a, 0x400ab6)
2017-11-28 17:25:38,425: [37716] m.c.executor:INFO: load state 3
2017-11-28 17:25:38,838: [37716] m.manticore:INFO: Generated testcase No. 0 - Program finished with exit status: 0
```

Reply:
I think this is okay, `exit(2)` is defined to not return.